libs/ledger-live-common/src/families/cosmos/js-signOperation.ts

```diff
@@ -1,21 +1,20 @@
-import type { CosmosAccount, Transaction } from "./types";
-import { defaultCosmosAPI } from "./api/Cosmos";
+import type { Transaction } from "./types";
 import { Observable } from "rxjs";
 import { withDevice } from "../../hw/deviceAccess";
 import { encodeOperationId } from "../../operation";
-import Cosmos from "@ledgerhq/hw-app-cosmos";
-import { AminoTypes } from "@cosmjs/stargate";
+import { LedgerSigner } from "@cosmjs/ledger-amino";
```

Comment:
What's the reason for using `LedgerSigner` instead of `hw-app-cosmos`? Is it because `hw-app-cosmos` only supports ATOM? If `hw-app-cosmos` works we should use it, otherwise maybe we should plan a task for later, to extend it to the same level of support as `LedgerSigner`.

Also, unrelated to this PR but, I think amino is a legacy format, the new format is protobuf and the Cosmos ecosystem will probably remove support for amino someday...